HTML-exercise/style.css

```diff
@@ -0,0 +1,495 @@
+a {text-decoration: none;}
+h3 {
+  font-size: 2rem;
+  font-weight: 300;
+  color: #333333;
+}
+html {
+  width: 100vw;
+
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+}
+::-webkit-scrollbar{ display: none; }
+
+body{ position: relative;}
+footer{ background-color: #fbb03b;}
+header{
+  width:100%;
+  height: 3rem;
+
+  position: absolute;
+  z-index: 100;
+  top: 3rem;
+
+  font-weight: 700;
+  font-size: 1.25rem;
+}
+input{
+  height: 3rem;
+
+  padding: 1rem;
+
+  border-radius: 1.5rem;
+  border: none;
+  outline: none;
+  text-indent: 1rem;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.text-primary {color: #fbb03b;}
+.text-danger {color: #ff0000;}
+.text-dark {color: #333333;}
+.text-muted {color: #a8a8a8;}
+
+.container {
+  height: 100%;
+  width:100%;
+  max-width:1440px;
+
+  margin: 0 auto;
+  padding:0 144px;
+}
+.row{
+  margin:0 -.625rem;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+}
+.col {
+  display: flex;
+  flex: 1;
+  padding: 0 .625rem;
```

Comment:
Em thử dùng cái này để tránh lặp padding nhiều lần https://stackoverflow.com/questions/9836151/what-is-this-css-selector-class-span
=> tránh lặp code tối ưu đc size của file => web load nhanh hơn